WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -2324,6 +2324,20 @@ private GutenbergPropsBuilder getGutenbergPropsBuilder() {
         );
     }
 
+    /**
+     * Checks if the theme supports the new gallery block with image blocks.
+     * Note that if the editor theme has not been initialized (usually on the first app run)
+     * the value returned is null and the `unstable_gallery_with_image_blocks` analytics property will not be reported.
+     * @return true if the the supports the new gallery block with image blocks or null if the theme is not initialized.
+     */
+    private Boolean themeSupportsGalleryWithImageBlocks() {
+        EditorTheme editorTheme = mEditorThemeStore.getEditorThemeForSite(mSite);
+        if (editorTheme == null) {
+            return null;
+        }
+        return editorTheme.getThemeSupport().getGalleryWithImageBlocks();
```

Comment:
Question from a newbie to Kotlin, just getting curious: I was wondering if there were a way in Kotlin to do "safe navigation" on nullable values (like `&.` in ruby or `?.` in Swift)… and found [this](https://kotlinlang.org/docs/null-safety.html#safe-calls).

So would this work too?
```suggestion
        EditorTheme editorTheme = mEditorThemeStore.getEditorThemeForSite(mSite);
        return editorTheme?.getThemeSupport().getGalleryWithImageBlocks();
```

Reply:
Yep, that would work perfectly in Kotlin :)
Unfortunately `EditPostActivity` activity is still in Java :(

Reply:
🤦‍♂️ missed that, good point 😅